armi/physics/neutronics/fissionProductModel/tests/test_lumpedFissionProduct.py

```diff
@@ -98,55 +101,26 @@ def setUp(self):
             io.StringIO(LFP_TEXT)
         )
 
-    def test_setGasRemovedFrac(self):
-        """Test of the set gas removal fraction"""
-        lfp = self.fpd.createSingleLFPFromFile("LFP38")
-        xe135 = nuclideBases.fromName("XE135")
-        gas1 = lfp[xe135]
-        lfp.setGasRemovedFrac(0.25)
-        gas2 = lfp[xe135]
-        self.assertAlmostEqual(gas1 * 0.75, gas2)
-
     def test_getYield(self):
         """Test of the yield of a fission product"""
         xe135 = nuclideBases.fromName("XE135")
         lfp = self.fpd.createSingleLFPFromFile("LFP39")
-        lfp[xe135] = 3
+        lfp[xe135] = 1.2
```

Comment:
This is changed since I put a validator on the yields that they must be within `[0.0, 2.0]`.

Reply:
We talked offline about the upper bound on the yields being 2, and about potentially removing that limitation.

Reply:
I tied this to a constant that was fixed to 2 in the code so at least its tied to another assumption.